app/views/shared/_display_ad.html.erb

```diff
@@ -1,55 +1,31 @@
-<div class="crayons-card crayons-card--secondary crayons-sponsorship"
-  data-display-unit data-id="<%= display_ad.id %>"
-  data-category-click="<%= DisplayAdEvent::CATEGORY_CLICK %>"
-  data-category-impression="<%= DisplayAdEvent::CATEGORY_IMPRESSION %>"
-  data-context-type="<%= data_context_type %>"
-  data-type-of="<%= display_ad.type_of %>">
-  <div class="crayons-sponsorship__header relative">
-    <% if display_ad.organization_id? && (display_ad.type_of == "community" || display_ad.type_of == "external") %>
-      <div class="flex align-center">
-        <img width="24" height="24" class="radius-default crayons-sponsorship__image" src="<%= display_ad.organization.profile_image_url_for(length: 64) %>" alt="profile" loading="lazy" />
-        <div class="crayons-sponsorship__title ml-2 fs-s fw-medium"><%= display_ad.organization.name %></div>
-        <% if display_ad.type_of == "external" %>
-          <span class="crayons-sponsorship__indicator c-indicator c-indicator--subtle c-indicator--round fs-xs fw-medium ml-2 py-1 px-2"> <%= I18n.t("display_ad.promoted") %></span>
-        <% end %>
+<% if display_ad.placement_area === "feed_first" || display_ad.placement_area === "feed_second" || display_ad.placement_area === "feed_third" %>
```

Comment:
As a non-blocking suggestion, this could maybe be written out as `display_ad.placement_area.start_with?("feed_")` which might be just a bit cleaner.

Reply:
Applied this change. Thanks.